gui/wxpython/lmgr/layertree.py

```diff
@@ -1999,7 +1999,7 @@ def OnDrop(self, dropTarget, dragItem):
         # save everything associated with item to drag
         try:
             old = dragItem  # make sure this member exists
-        except:
+        except Exception:
```

Comment:
Weird, I don't see how this could throw any exception.

Reply:
Yeah, I noticed that, however I wasn't sure why exactly that `try` block was there hence just put a general `Exception`. I can remove that entire `try` block alternatively

Reply:
I would probably remove the try except here.